docs/conf.py

```diff
@@ -1,3 +1,4 @@
+# noqa
```

Comment:
This noqa disables all the Ruff checks for this file, doesn't it?

Reply:
as far as I know, no it only disables it for this first line which is a module level docstring.

To disable the entire file you need # ruff: noqa at the top of file (see [here](https://github.com/astral-sh/ruff/pull/2978))